libmilter.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: libmilter
-  version: 1.0.2
+  version: 1.0.3
```

Comment:
did not find where this version come from, alpine does the same, so since this is a patch version of sendmail we do a patch bump here as well

Reply:
I see 1.0.2 in Alpine edge: https://pkgs.alpinelinux.org/package/edge/main/x86_64/libmilter

Where do you see Alpine calling this 1.0.3 ?

Reply:
i mean they are using the same version.
but they did not update to `sendmail.8.17.2` it is still on `sendmail.8.17.1`